scss/_offcanvas.scss

```diff
@@ -41,7 +44,7 @@
       background-clip: padding-box;
       outline: 0;
       @include box-shadow(var(--#{$prefix}offcanvas-box-shadow));
-      @include transition(transform $offcanvas-transition-duration ease-in-out);
+      @include transition(transform var(--#{$prefix}offcanvas-transition-duration) ease-in-out);
```

Comment:
This should probably be a variable for the entire property value, not just the duration.

```css
--bs-offcanvas-transition: transform #{$offcanvas-transition-duration} ease-in-out);
```